torch/_inductor/codecache.py

```diff
@@ -1184,7 +1184,9 @@ def cpp_wrapper_flags() -> str:
 
 def optimization_flags() -> str:
     base_flags = "-O0 -g" if config.aot_inductor.debug_compile else "-O3 -DNDEBUG"
-    base_flags += " -ffast-math -fno-finite-math-only"
+    # -ffast-math without -funsafe-math-optimizations
+    base_flags += " -fno-math-errno -ffinite-math-only -fno-rounding-math -fno-signaling-nans -fcx-limited-range"
+    base_flags += " -fno-finite-math-only"
```

Comment:
You don't need to specify `-ffinite-math-only` and `-fno-finite-math-only` at the same time. Just remove both.